docs/gallery.yml

```diff
@@ -1,20 +1,20 @@
-- title: A Gentle Introduction to xCDAT (Xarray Climate Data Analysis Tools)
+- title: Introduction to xCDAT
   path: examples/introduction-to-xcdat.ipynb
   thumbnail: _static/xcdat-logo.png
 
-- title: General Dataset Utilities
+- title: xCDAT Utilities
   path: examples/general-utilities.ipynb
   thumbnail: _static/dataset-diagram.webp
 
-- title: Calculate Geospatial Weighted Averages from Monthly Time Series
+- title: Spatial Averaging
```

Comment:
I will update the notebook title, too, so that this is reflected in the table of contents link.

Reply:
Sounds good